br/pkg/lightning/backend/local/duplicate.go

```diff
@@ -550,6 +562,11 @@ func (m *DupeDetector) RecordIndexConflictError(ctx context.Context, stream DupK
 		if err != nil {
 			return errors.Trace(err)
 		}
+
+		if algorithm == config.DupeResAlgErr {
```

Comment:
we also have variable `indexInfo`, no need to find the index info again inside `NewErrFoundConflictRecords`